### DISCUSSION TITLE
How to get clipped text as bullet or numbered list

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#100] [Q&A]
Is it possible to get clipped text as bullet or numbered list especially if we are clipping multiple lines at a time

**Comment by buggyj** [comment]
I don't understand... maybe you could show an example?

**Reply by arunnbabu81** [reply]
It is possible to select non contiguous text by keep pressing control or command key while selecting. Is it possible to get each of these selected text as bullet list or numbered list using wikitext formatting like shown below

- one
- Two
- Three

**Comment by buggyj** [comment]
When I try to select multiple parts of a text using the crtl key, firefox joins these parts together as a single paragraph, so tiddlyclip sees this as a single paragraph. So no, it is not possible to do what you want with a single clip.

**Reply by buggyj** [reply]
you could use text2note with each individual selection and then somehow format the results within you tiddlywiki.
